apps/site/assets/ts/helpers/service.ts

```diff
@@ -1,35 +1,14 @@
-import { Service, ServiceWithServiceDate } from "../__v3api";
-
-const monthIntegerToString = (month: MonthInteger): string =>
-  [
-    "January",
-    "February",
-    "March",
-    "April",
-    "May",
-    "June",
-    "July",
-    "August",
-    "September",
-    "October",
-    "November",
-    "December"
-  ][month];
-
-const formattedDate = (date: Date): string =>
-  `${monthIntegerToString(date.getUTCMonth())} ${date.getUTCDate()}`;
-
-const holidayDate = (service: Service): string => {
-  const note = service.added_dates_notes[service.start_date];
-  return note || `on ${formattedDate(new Date(service.start_date))}`;
-};
+import { Service } from "../__v3api";
+import { shortDate } from "./date";
+
+const formattedDate = (date: Date): string => shortDate(date);
```

Comment:
Could we replace instances of `formattedDate` with `shortDate` and remove this delegation?

Reply:
Done